site/app/templates/forum/CreatePost.twig

```diff
@@ -130,3 +134,9 @@
 
     </form>
 {% endif %}
+
+<script>
+    $(document).ready(function() {
+        checkIfAnnounced({{ activeThread.id }});
+    });
+</script>
```

Comment:
why are we fetching this information via AJAX after page load, as opposed to just passing it into the twig template from the view?